### DISCUSSION TITLE
Help with Free Account

### DISCUSSION THREAD

**Discussion by George-Brendan-Doyle** [emailjs-com/help#85] [Q&A]
Hi I am testing the emailjs system and using the FREE Account.
My problem is that I cannot set the receivers email address.  Is this a limitation of the FREE service? and default template.
I receive the emails all the time

var param1 = jQuery.QueryString["newuser"];
var param2 = jQuery.QueryString["guid"];
var data = "?newuser=" + param1 + "&guid=" + param2;

if (param1 != undefined && param2 != undefined) {
    // Send email to param1
    var templateParams = {
        from_name: 'georgebrendandoyle@gmail.com',
        user_email: param1,
        to_name: param1,
        CC: param1,
        message: 'Please Confirm Registration by clicking the link ' + 'http://www.georgebrendandoyle.somee.com/ConfirmEmail' + data
    };
    emailjs.send('myservice', 'mytemplate', templateParams)
        .then(function (response) {
            alert('SUCCESS! ' + response.status + ' ' + response.text);
        }, function (error) {
            alert('FAILED...'+ error);
        });

    //alert(data);
}

**Comment by xr0master** [comment]
The recipient's address is configured in the template. In the "Email To" field.
